vmngclient/api/template_api.py

```diff
@@ -186,7 +186,7 @@ def get_device_specific_variables(name: str):
         endpoint = "/dataservice/template/device/config/attachfeature"
         logger.info(f"Attaching a template: {name} to the device: {device.hostname}.")
         response = self.session.post(url=endpoint, json=payload).json()
-        task = wait_for_completed(session=self.session, action_id=response["id"])
+        task = wait_for_completed(session=self.session, action_id=response["id"])[0]
```

Comment:
```suggestion
        task = wait_for_completed(session=self.session, action_id=response["id"]).single_or_default()
```

Reply:
OK, at first I was confused with .singe or default, but it also made sense at the same time.
Now that I see it in code it it's obvious...
`.single_or_default() == [0]`